packages/react-renderer-demo/src/pages/importing.md

```diff
@@ -0,0 +1,41 @@
+import DocPage from '@docs/doc-page';
+
+<DocPage>
+
+# Importing
+
+Data Driven Forms is built in three module formats: UMD, CJS and ESM.
+
+## UMD
```

Comment:
I would definitely stress that UMD is there just as a fallback option and should be used if you are building inside CJS or ESM env.